### PR TITLE
docs: update readme, add license and init contributing

### PR DESCRIPTION
**Pull request by ramonxm**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [X] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+```sh
+git clone https://github.com/<your username>/ed-ui.git
+cd ed-ui
+git remote add upstream https://github.com/estartando-devs/ed-ui.git
+```
+
+<!-- #default-branch-switch -->
+
+3. Synchronize your local `master` branch with the upstream one:
```
Comment (jonataspinto) [inline .github/CONTRIBUTING.md:44]: main here?

**Review by jonataspinto — APPROVED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (ramonxm) [reply .github/CONTRIBUTING.md:44]: exists `main` and `master`, which one do we use? or what is their use?

Comment (jonataspinto) [reply .github/CONTRIBUTING.md:44]: my preference is `main`.  😸

Comment (ramonxm) [reply .github/CONTRIBUTING.md:44]: I'll delete the `master` branch then, ok?

Comment (jonataspinto) [reply .github/CONTRIBUTING.md:44]: ok!

**Pull request merged**